pages/pipelines/notifications.md

```diff
@@ -154,6 +154,9 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+> 📘Step-level vs build-level
+> A step-level notify step will ignore the requirements of a build-level notification. If a build-level notification condition is that it runs only on `main`, a step-level notification will run on all branches if no branch conditional is set.
```

Comment:
Some minor tweaks to the callout intro line (so far) ...

```suggestion
> 📘 Step-level vs build-level notifications
> A step-level notify step will ignore the requirements of a build-level notification. If a build-level notification condition is that it runs only on `main`, a step-level notification will run on all branches if no branch conditional is set.
```

> ... if no branch conditional is set

This clause sounds a little incomplete, perhaps because 'set' (in this context) is a transitive verb and hence, the clause's object is missing.

Does this mean '... if no branch conditional has been set on this step-level notification' ?